install/linux/vendor-pymupdf.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+import argparse
+import subprocess
+import sys
+from pathlib import Path
+
+DZ_VENDOR_DIR = Path("./dangerzone/vendor")
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "--dest",
+        default=DZ_VENDOR_DIR,
+        help="The destination directory for the vendored packages (default: ./dangerzone/vendor)",
+    )
+    args = parser.parse_args()
+
+    print(">>> Getting PyMuPDF deps as requirements.txt", file=sys.stderr)
```

Comment:
Why using `stderr` here? 🤔

Reply:
Well, I prefer to print log message in `stderr`, that's why I mention it explicitly. But I can replace this with logigng statements (which also log in `stderr`), if it feels ugly.

Reply:
I think I would prefer it if we were to use the logger functions of python instead. By default, the [`LastResort` handler](https://docs.python.org/3/library/logging.html#logging.lastResort) outputs to `stderr`, but it's also possible to configure it to behave differently.